Source/ASPagerNode.m

```diff
@@ -66,6 +66,7 @@ - (instancetype)init
 - (instancetype)initWithCollectionViewLayout:(ASPagerFlowLayout *)flowLayout;
 {
   ASDisplayNodeAssert([flowLayout isKindOfClass:[ASPagerFlowLayout class]], @"ASPagerNode requires a flow layout.");
+  ASDisplayNodeAssertTrue(flowLayout.scrollDirection == UICollectionViewScrollDirectionHorizontal);
```

Comment:
The existing implementation of this class (e.g `-currentPageIndex` and `- (void)scrollToPageAtIndex: animated:`) assumes a horizontal scroll direction. Let's formalize that.